yarn.lock

```diff
@@ -4563,133 +4024,133 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@react-spring/animated@npm:~10.0.0":
-  version: 10.0.0
-  resolution: "@react-spring/animated@npm:10.0.0"
+"@react-spring/animated@npm:~10.0.3":
+  version: 10.0.3
+  resolution: "@react-spring/animated@npm:10.0.3"
```

Comment:
This is what's causing playwright to hang, the tests won't pass with this version of `react-spring`. I hit the issue in #5865 and chose to defer it to another PR because I couldn't get it working in a nice way. It messes with the animation when dragging the navbar.

We need to be careful regenerating the lock file because it will bump any dependencies matched in the package.json ranges. This shouldn't be a problem, but we've not been on top of pinning versions and upgrading so minor bumps like these can cause issues.